.github/ISSUE_TEMPLATE/giskard_bug_template.yaml

```diff
@@ -0,0 +1,136 @@
+name: Giskard Bug Template
+description: Use this template to report any bugs
+body:
+  - type: dropdown
+    id: issue-type
+    attributes:
+      label: Issue Type
+      description: What type of issue would you like to report?
+      multiple: false
+      options:
+        - Bug
+        - Build/Install
+        - Performance
+        - Documentation Bug
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/Giskard-AI/giskard/blob/main/ISSUES.md),we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
+
+  # - type: markdown
+  #   attributes:
+      # value: | 
+      #   You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)  You can also obtain the TensorFlow version with: <br> 1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` <br>2. TF 2.0: `python -c "import giskard ; print(tf.version.GIT_VERSION, tf.version.VERSION)"`
+
+  - type: dropdown
+    id: source
+    attributes:
+      label: Source
+      description: Giskard installed from
+      options:
+        - source
+        - binary
+    validations:
+      required: true
+  - type: input
+    id: giskardversion
+    attributes:
+      label: Giskard Version
+      description: 
+      placeholder: ex,. giskard 1.3
+    validations:
+      required: true
+  - type: dropdown
+    id: Code
+    attributes:
+      label: Custom Code
```

Comment:
I'm not sure what is meant by Custom Code here?

Reply:
Our users have the ability to write custom tests. Here custom code means if the code raises the bug when integrating with the custom code. I ll add a description to help users understand better.

Reply:
Actually I don't know if having `Code` section is very useful since there's also `code-to-reproduce`. Since "Custom Code" is pretty vague it'd rather remove it